### DISCUSSION TITLE
How can I add snaps to the applications menu on rofi?

### DISCUSSION THREAD

**Discussion by Torelli** [prasanthrangan/hyprdots#1951] [Q&A] [answered]
I installed the spotify snap but It doesn't appear on the rofi app selection.

**Comment by kRHYME7** [comment]
When installing snap have you tried reading the wiki?

https://wiki.archlinux.org/title/Snap

**Reply by Torelli** [reply]
Yes I did, and the only way for me to run my snap applications is using the ` snap run <app_name>`. They don't appear in the app list on rofi select

**Reply by kRHYME7** [reply, marked as answer]
![Screenshot_2024-10-31-08-01-16-806_org.mozilla.firefox-edit.jpg](https://github.com/user-attachments/assets/498105c0-2e3c-4c05-86fa-7e01fadc73d7)
I stopped using snaps so I can't help this much. But how about checking out this said script

Rofi searches the .desktop file usually in ` ./share/application/*.desktop `. Therefore check if snap populates the *.desktop files to those directories

**Reply by Torelli** [reply]
It worked, I just had to reboot my system. Thank you!

**Reply by kRHYME7** [reply]
Are the boot duration of snaps still slow? or there were some improvements?

**Reply by Torelli** [reply]
In my opinion I think that is manageable, It's not as fast as a native app, but it doesn't annoy me
